experimental/emulated_vault/opt/emulated-vault/conf/cfg.ini

```diff
@@ -0,0 +1,13 @@
+[general]
```

Comment:
Love to see this format being adopted :+1: 
but you're missing a license header in this file